Assets/MoveBase.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public abstract class MoveBase : MonoBehaviour
+{
+    public enum EMoveType
+    {
+        None = 0,
+        Left,
+        Right,
+        Up,
+        Down,
+    }
+
+    public enum EBlockLayerType
+    {
+        Player = 7,
+        Enemy = 8,
+        Ground = 6,
+    }
+
+    [SerializeField]
+    protected EMoveType eMoveType = EMoveType.None;
```

Comment:
```suggestion
    [SerializeField]
    protected EMoveType _eMoveType = EMoveType.None;
```

protected変数名の _ を統一して欲しいです。

Reply:
ご指摘ありがとうございます。　他の名称にあわせて、先頭に_をつけました。　ご確認よろしくです。